<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dhaval" failures="2" tests="2" name="Test" time="66.657" errors="0" timestamp="2023-02-03T11:14:40 IST">
  <testcase classname="com.orderocks.tests.OrderTest" name="verifyLogin" time="0.496">
    <failure type="java.lang.AssertionError" message="expected [Let&amp;apos;s get you started...] but found [Let&amp;apos;s get you started]">
      <![CDATA[java.lang.AssertionError: expected [Let's get you started...] but found [Let's get you started]
at com.orderocks.tests.OrderTest.verifyLogin(OrderTest.java:91)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogin -->
  <testcase classname="com.orderocks.tests.OrderTest" name="verifyPlaceOrder" time="0.09">
    <failure type="java.lang.AssertionError" message="expected [Let&amp;apos;s get you started...] but found [Let&amp;apos;s get you started]">
      <![CDATA[java.lang.AssertionError: expected [Let's get you started...] but found [Let's get you started]
at com.orderocks.tests.OrderTest.verifyPlaceOrder(OrderTest.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyPlaceOrder -->
</testsuite> <!-- Test -->
